lib/services/redfish-validator-service.js

```diff
@@ -207,6 +211,34 @@ function redfishValidatorFactory(
         return Promise.reject(new Errors.NotFoundError ('Message registry not found'));
 
     };
+    RedfishValidator.prototype.getVendorNameById = function(identifier) {
```

Comment:
It is not suitable to put this function in `RedfishValidator`. `RedfishValidator` is a module which designed manages the redfish json schema validation. Actually this function has nothing relationship with schema and validation.  So please move it out.